### ISSUE TITLE
Escape underscores in grouping variables in lmer() 

### ISSUE THREAD

**Issue by fschaffner** [leifeld/texreg#137]
I run a model with the lme4 package and have a grouping variable with an underscore. In the output generated with texreg() the underscore in "species_variable" needs to be escaped, but it is currently not. Reproducible example:

```
# Reprex
iris$species_variable <- iris$Species
fit <- lme4::lmer(Sepal.Length ~ Sepal.Width + (1 | species_variable), data = iris)
texreg(fit)
```
texreg(fit) currently produces this:

```
Num. groups: species_variable     & $3$          \\
Var: species_variable (Intercept) & $1.02$       \\
```

And this is what it should look like: 

```
Num. groups: species\_variable     & $3$          \\
Var: species\_variable (Intercept) & $1.02$       \\
```

Note: This query may be related to issues #77 and #86.

**Comment by czucca** [comment]
This issue is fixed in the GitHub update version, not on CRAN. So if you are using texreg 1.36.23 download the update version. If you are already using the updated version let us know.

**Comment by fschaffner** [comment]
I have tried both the CRAN version and the development version on GitHub (texreg_1.36.24) but both produce the same output. 

Note that I am talking about the grouping variable and not the main part of the model. Here's an updated version of my reprex: 

```
iris$species_variable <- iris$Species
iris$sepal_variable <- iris$Sepal.Width
fit <- lme4::lmer(Sepal.Length ~ sepal_variable + (1 | species_variable), data = iris)
texreg(fit)
```
This produces the following output: 


```
\begin{table}
\begin{center}
\begin{tabular}{l c}
\hline
 & Model 1 \\
\hline
(Intercept)                       & $3.41^{***}$ \\
                                  & $(0.67)$     \\
sepal\_variable                   & $0.80^{***}$ \\
                                  & $(0.11)$     \\
\hline
AIC                               & $202.64$     \\
BIC                               & $214.68$     \\
Log Likelihood                    & $-97.32$     \\
Num. obs.                         & $150$        \\
Num. groups: species_variable     & $3$          \\
Var: species_variable (Intercept) & $1.02$       \\
Var: Residual                     & $0.19$       \\
\hline
\multicolumn{2}{l}{\scriptsize{$^{***}p<0.001$; $^{**}p<0.01$; $^{*}p<0.05$}}
\end{tabular}
\caption{Statistical models}
\label{table:coefficients}
\end{center}
\end{table}
```

Note that sepal\\_variable is correctly escaped but `Num. groups: species_variable` and `Var: species_variable (Intercept)` are not.

**Comment by czucca** [comment]
ok, we did not fix the grouping. Feel free to send PR. Otherwise we'll get to this asap.

**Comment by ashirwad** [comment]
@czucca , any updates on this? This feature still seems to be broken.

**Comment by czucca** [comment]
Hi @ashirwad, not yet. We'll get to this when bigger upgrades will be done. Sorry!

**Comment by ashirwad** [comment]
@czucca, no worries! Just keep us posted. :)

**Comment by leifeld** [comment]
Should be fixed in commit 42108ac12103f23b6769e38bb6cdbfd4745bbe98. Please re-open and let us know if there are any further issues with this. Thanks.
